graphql/queries/addModule.ts

```diff
@@ -4,6 +4,11 @@ const ADD_MODULE = gql`
   mutation addModule($content: String!, $lessonId: Int!, $name: String!) {
     addModule(content: $content, lessonId: $lessonId, name: $name) {
       id
+      name
```

Comment:
We already have these in the frontend data the moment the mutation `addModule` is ran. So, I imagine something like this:

```js
const { data, error } = addModule({ content, lessonId, name })

if (data && data?.id) setState(content, lessonId, name)

if (error) showError(true)
```